this testsuite relies on a network resource
that is not available anymore
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -1080,7 +1080,6 @@
 DDSTESTSUITE = $(srcdir)/DDSTest
 EXPRTESTSUITE = $(srcdir)/EXPRTest
 DMRTESTSUITE = $(srcdir)/DMRTest
-GETDAPTESTSUITE = $(srcdir)/getdapTest
 AUTOTEST = $(AUTOM4TE) --language=autotest
 all: all-am
 
@@ -1847,7 +1846,6 @@
 	$(SHELL) '$(DDSTESTSUITE)' $(TESTSUITEFLAGS)
 	$(SHELL) '$(EXPRTESTSUITE)' $(TESTSUITEFLAGS)
 	$(SHELL) '$(DMRTESTSUITE)' $(TESTSUITEFLAGS)
-	$(SHELL) '$(GETDAPTESTSUITE)' $(TESTSUITEFLAGS)
 
 # Make sure there are no spaces after the DASTESTSUITE names. jhrg 3/16/15
 # Delete the scripts here and not using CLEANFILES because this target is
